pkg/standalone/list.go

```diff
@@ -156,3 +160,16 @@ func getIntArg(argMap map[string]string, argKey string, argDef int) int {
 	}
 	return argDef
 }
+
+// GetCLIPIDCountMap returns a map of CLI PIDs to number of apps started with it.
+func GetCLIPIDCountMap() map[int]int {
+	cliPIDCountMap := make(map[int]int)
```

Comment:
move to next line and change to 
```suggestion
	cliPIDCountMap := make(map[int]int, len(apps))
```